sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/PropertiesAccessCorrectField.cs

```diff
@@ -727,6 +727,7 @@ public List<int> Data
         }
     }
 
+#if NETFRAMEWORK // System.Windows.Controls.Primitives.ButtonBase is in a different assembly in NETCOREAPP
```

Comment:
We could add a link to https://github.com/SonarSource/sonar-dotnet/issues/3426 since it's windows forms related. What do you think?